src/index.html

```diff
@@ -676,10 +679,9 @@ <h5>Acknowledgement</h5>
                                         <a class="accordion-title"><h5>Can I download accessioned variants from the EVA?</h5></a>
                                         <div class="accordion-content" data-tab-content>
                                             <p>
-                                                Yes. After an rs build the EVA will provide a VCF dump for each species/assembly combination.
-                                                These files shall be in the same style as the current VCFs that dbSNP generates (i.e. contain the rs ID positional information, but not the genotypic nor annotation data).
-                                                Furthermore we are developing a 'VCF Dumper' tool which shall allow users to generate a custom VCF file based on filtering options available at the
-                                                <a href="?Variant%20Browser&species=btaurus_umd31&selectFilter=region&studies=PRJEB6119%2CPRJEB7061%2CPRJEB14878&region=1%3A3000000-3100000">EVA Variant Browser</a>.
+                                                Yes. Each non-human study archived in EVA will contain a VCF file with the accessioned variants (SS ids).
+                                                After an rs build the EVA will provide a VCF dump for each species/assembly combination.

```

Comment:
Maybe:
```suggestion
                                                After an RS release the EVA will provide a VCF dump for each species/assembly combination.

```
?
I don't know if we use the "build" language anywhere, but if people will be familiar with it from dbSNP it's probably fine.